go.mod

```diff
@@ -7,12 +7,12 @@ require (
 	github.com/hashicorp/golang-lru v0.6.0
 	go.uber.org/zap v1.24.0
 	istio.io/api v0.0.0-20221212180111-09895d694711
-	k8s.io/api v0.25.6
-	k8s.io/apimachinery v0.25.6
+	k8s.io/api v0.26.2
+	k8s.io/apimachinery v0.26.2
 	k8s.io/client-go v0.25.6
```

Comment:
can we bump here as well to 0.26.2? to match the other k8s components

Reply:
unfortunately not, we have to stay in sync with knative/pkg for the k8s.io/client-go

would you prefer i keep all of it consistent? ie. all on 0.25.6?

Reply:
yep then if possible move all to 0.25.x